Marlin/G26_Mesh_Validation_Tool.cpp

```diff
@@ -317,6 +299,27 @@
         }
 
         for (tmp = start_angle; tmp < end_angle - 0.1; tmp += 30.0) {
+
+          // this sequence to detect an ubl_lcd_clicked() debounce it and leave if it is
+          // a Press and Hold is repeated in a lot of places (including ubl_G29.cpp).   This
+          // should be redone and compressed.
+          if (ubl_lcd_clicked()) {              // Check if the user wants to stop the Mesh Validation
+            #if ENABLED(ULTRA_LCD)
+              lcd_setstatuspgm(PSTR("Mesh Validation Stopped."), 99);
+              lcd_quick_feedback();
+            #endif
+            while (!ubl_lcd_clicked()) {         // Wait until the user is done pressing the
+              idle();                            // Encoder Wheel if that is why we are leaving
+              lcd_reset_alert_level();
+              lcd_setstatuspgm(PSTR(""));
+            }
+            while (ubl_lcd_clicked()) {          // Wait until the user is done pressing the
+              idle();                            // Encoder Wheel if that is why we are leaving
+              lcd_setstatuspgm(PSTR("Unpress Wheel"), 99);
```

Comment:
Users of RepRapWorld Keypad and Cartesio UI don't have a "wheel." And my LCDs have a knob.